internal/controllers/topology/cluster/patches/engine.go

```diff
@@ -431,3 +483,15 @@ func updateDesiredState(ctx context.Context, req *runtimehooksv1.GeneratePatches
 
 	return nil
 }
+
+func definitionsForPatch(blueprint *scope.ClusterBlueprint, definitionFrom string) map[string]bool {
```

Comment:
```suggestion
// definitionsForPatch returns a set which contains all variables defined by "definitionFrom".
func definitionsForPatch(blueprint *scope.ClusterBlueprint, definitionFrom string) map[string]bool {
```
nit: or something similar